design-at-cornell/src/components/HomepageStyles.ts

```diff
@@ -97,3 +116,13 @@ export const ExploreSubtitle = styled.div`
   width: 100%;
   cursor: pointer;
 `;
+
+export const Grid = styled.div`
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  height: fit-content;
+  margin-top: 90px;
+  margin-left: 40px;
```

Comment:
Since we want to delete `margin-right` in `ElementContainer`, we would also need to delete `margin-left: 40px;` here.